components/select/demo/custom-tag-render.md

```diff
@@ -0,0 +1,54 @@
+---
+order: 24
+title:
+  zh-CN: 自定义tag内容render
+  en-US: Custom Tag Render
+---
+
+## zh-CN
+
+允许自定义选择标签的样式
+
+## en-US
+
+Allows for custom render of tags
+
+```jsx
+import { Select, Tag } from 'antd';
+
+const OPTIONS = ['Hello', 'World', '42'];
```

Comment:
OPTIONS = ['gold', 'cyan']

Reply:
and follow `tagRender` just reuse the value.

Reply:
> and follow `tagRender` just reuse the value.

If I do that without extra checks any other tags will essentially be transparent. As long as you're OK with that.

Reply:
We can just simplify demo for `multiple` instead:
```jsx
<Select
    mode="multiple"
    tagRender={tagRender}
    defaultValue={OPTIONS}
    style={{ width: '100%' }}
    options={...}
/>
```

PS: Not `Options` in children. `options` is a prop.

Reply:
I've now switched over to "multiple" and aligned the demo to the "big-data" one for options.